build_daemon/lib/data/server_log.dart

```diff
@@ -4,14 +4,102 @@
 
 import 'package:built_value/built_value.dart';
 import 'package:built_value/serializer.dart';
+import 'package:logging/logging.dart' as logging;
 
 part 'server_log.g.dart';
 
+enum Level {
+  finest,
```

Comment:
I'm not sure if we want to spread the choices of `package:logging` here... I would be in favor of shortening this to `fine, info, warning, severe`

Reply:
We expose all these levels to builder authors though, so we should retain them imo.

These aren't just the opinions of `package:logging` either, they are at least standardized across Java/Dart :D.